ilastik/config.py

```diff
@@ -87,6 +89,12 @@ def init_ilastik_config(userConfig=None):
     if userConfig is not None and not os.path.exists(userConfig):
         raise Exception("ilastik config file does not exist: {}".format(userConfig))
 
+    # Move default config to the new location.
+    oldConfigPath = os.path.expanduser("~/.ilastikrc")
+    if userConfig is None and os.path.exists(oldConfigPath) and not os.path.exists(CONFIG_PATH):
+        os.makedirs(os.path.dirname(CONFIG_PATH), exist_ok=True)
+        shutil.move(oldConfigPath, CONFIG_PATH)
+
```

Comment:
Since we're here anyway, what do you think about moving the call to `init_ilastik_config()` into `app.py`? I find it unnerving that it just happens silently on module `import`

Reply:
Probably a good idea, but I'm afraid this will break other code that depends on module initialisation.

Reply:
i agree that this is a good idea, so it should get an issue and be tackled in a separate PR